charts/ordapi/templates/deployment.yaml

```diff
@@ -60,6 +60,71 @@ spec:
                   key: {{ .Values.serviceDependencies.oidc.baseUriConfig.key | quote }}
             - name: ALLOWINVALIDCERTIFICATE
               value: {{ .Values.allowInvalidCertificate | quote }}
+            {{- with .Values.serviceDependencies.email }}
+            - name: SMTPSERVER__HOST
```

Comment:
Please can you add these settings to appsettings.json, so it's clear that they can be set, with sensible defaults?

Reply:
They are in a PR waiting for review & hopefully soon to be merged in, I just split the tasks out.